src/components/Ui/Button.vue

```diff
@@ -12,7 +12,7 @@ defineProps({
 <template>
   <button
     :type="type || 'button'"
-    class="button px-[24px]"
+    class="button px-3"
```

Comment:
Ah alright, to me it seemed like a bug since padding was inconsistent between inputs and buttons leading to malaligned forms:

![image](https://user-images.githubusercontent.com/524089/134862145-b9399902-b43a-4a4b-a481-8e3aab57ec69.png)

It think all labels should be left aligned here. The top two inputs internally use the Button component, but I can try to change the padding only in this scenario. Or do you just wanna keep it like it is?

Reply:
If you use `UiInput` component you should have proper spacing, this is what we do on the space settings page here:
https://snapshot.org/#/fabien.eth/settings
![image](https://user-images.githubusercontent.com/16245250/134890081-6607d58f-4be0-45f5-8516-a15f748251cb.png)

Reply:
By the way it would be great if we use the same casing across all the app, on Snapshot we use "Sentence case", i would recommend to change the labels from lowercase or Title Case like "type", "asset", "Transaction Batch" etc to Sentence case like "Type", "Asset", "Transaction batch" etc